packages/cli/src/lib/machine/drivers/fake/index.ts

```diff
@@ -1,107 +0,0 @@
-import { Flags, Interfaces } from '@oclif/core'
-import { asyncMap } from 'iter-tools-es'
-import { SSHKeyConfig } from '../../../ssh/keypair'
-import { Machine, MachineDriver } from '../../driver'
-import { MachineCreationDriver, MachineCreationDriverFactory, MachineDriverFactory } from '../../driver/driver'
-
-const fakeMachine: Machine = {
```

Comment:
I think maybe we should still have fake driver in core. (since it doesn't have dependency, it should be a problem)

Reply:
Added the code, but I'm not sure it's useful. Its only purpose was to demo multiple drivers when we only had lightsail.

Reply:
Strike that - adding it brings back `@oclif/core` because of the flags definitions